.editorconfig

```diff
@@ -13,6 +13,9 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+# Spell checker configuration
```

Comment:
I had no clue this was a thing :D

Reply:
It's relatively new. Have wanted something like this in VS for ages.

The "ignore spelling" codefix in VS will automatically add to this file.